hive/src/test/java/com/netease/arctic/hive/catalog/TestArcticHiveCatalog.java

```diff
@@ -41,4 +56,61 @@ public static Object[] parameters() {
     return new Object[] {new HiveCatalogTestHelper(TableFormat.MIXED_HIVE, TEST_HMS.getHiveConf()),
                          new HiveCatalogTestHelper(TableFormat.ICEBERG, TEST_HMS.getHiveConf())};
   }
+
+  @Test
+  public void testArcticTableRootLocation() throws TException {
+    ArcticCatalog hiveCatalog = getCatalog();
+    // iceberg table doesn't support to set hive's property
+    if (hiveCatalog instanceof BasicIcebergCatalog) {
+      return;
+    }
+
+    String dbName = "arctic_location_test";
```

Comment:
Are the constant values in `TableTestHelper` for catalog/db/table okay for your test?

Reply:
> Are the constant values in `TableTestHelper` for catalog/db/table okay for your test?

 I referred TestBasicArcticCatalog:testCreateAndDropDatabase and felt there might be conflicts, so I think it's more appropriate to create a DB alone.